### PR TITLE
Reidcrews signoff location

### PR DESCRIPTION
**Pull request by reidcrews**

### CODE REVIEW

File: Documentation/Signoffs/ReidCrews-signoff-Location.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+* C14: The robot shall have a user manual that explains all functions and capabilities of the robot. 
+	
+    * This constraint was constructed to document the design process and motives of the location subsystem for future SECON teams.
+
+## Buildable Schematic 
+
+### Location Subsystem Schmatic
+
+![Alt text](https://github.com/lchapman42/Control-Sensing-Wireless-Charging-Robot/blob/reidcrews-signoff-Location/Documentation/Electrical/Schematics/Sources/Position/PositionSensorSchematic2.png)
```
Comment (n37chen) [inline Documentation/Signoffs/ReidCrews-signoff-Location.md:36]: How is the mega powered?

**Review by n37chen — CHANGES_REQUESTED** [review]
This is a relatively small signup. It may be good to combine it with other signups.

File: Documentation/Signoffs/ReidCrews-signoff-Location.md (hunk 1 above)
Comment (reidcrews) [reply Documentation/Signoffs/ReidCrews-signoff-Location.md:36]: By Power Distribution Bus,

Comment (reidcrews) [reply Documentation/Signoffs/ReidCrews-signoff-Location.md:36]: 5 V

**Review by n37chen — APPROVED** [review]

**Pull request merged**